html/pfappserver/root/static.alt/src/views/Auditing/_components/LiveLogView.vue

```diff
@@ -0,0 +1,533 @@
+              class="mb-3" size="sm"
+            />
+            <template v-if="lines > 0">
+              <template v-for="(children, scope) in scopes">
+                <small :key="scope" class="ml-1">{{ children.label }}</small>
+                <b-list-group :key="children" class="mt-1 mb-3">
+                  <template v-for="({ count, filter }, key) in children.values">
+                    <b-list-group-item :key="`${key}-${count}-${filter}`"
+                      href="#" class="cursor-pointer"
+                      :active="filter"
+                      :variant="(filter) ? 'primary' : 'light'"
+                      @click="toggleFilter(scope, key)"
+                      :title="(filter) ? $i18n.t('Click to disable filter') : $i18n.t('Click to enable filter')"
+                      v-b-tooltip.hover.top.d300
+                    >
+                      <template v-if="key">
+                        {{ key }}
+                      </template>
+                      <template v-else>
+                        <i>{{ $i18n.t('none') }}</i>
+                      </template>
+                      <b-badge class="float-right border text-secondary bg-light ml-1">{{ count }}</b-badge>
+                    </b-list-group-item>
+                  </template>
+                </b-list-group>
+              </template>
+            </template>
+          </div>
+        </b-col>
+        <b-col sm="9" class="pl-0" :class="`direction-${options.order}`">
+
+          <b-row class="mb-3" align-v="center" :class="(options.order === 'forward') ? 'mt-3' : 'mb-3'">
+            <b-col cols="auto" class="mr-auto d-inline">
+
+              <b-button v-if="isRunning && !isPaused"
+                variant="primary" class="ml-3" size="sm" @click="pauseSession()"
+              >
+                <icon name="pause" class="mr-2"></icon>
+                {{ $i18n.t('Pause') }}
+              </b-button>
+              <b-button v-if="isRunning && isPaused"
+                variant="primary" class="ml-3" size="sm" @click="unpauseSession()"
+              >
+                <icon name="play" class="mr-2"></icon>
+                {{ $i18n.t('Unpause') }}
+              </b-button>
+
+              <b-button-group class="mx-1 ml-3" size="sm" :disabled="!events || !events.length">
+                <b-button @click="copyEvents()" variant="outline-primary">{{ $t('Copy') }}</b-button>
+                <b-button @click="saveEvents()" variant="outline-primary">{{ $t('Save') }}</b-button>
+                <b-button @click="clearEvents()" variant="outline-danger">{{ $t('Clear') }}</b-button>
+              </b-button-group>
+
+              <b-button-group class="mx-1 ml-3" size="sm" :title="$i18n.t('Choose background')" v-b-tooltip.hover.top.d300>
+                <b-button @click="options.background = 'white'" :active="options.background === 'white'" variant="outline-dark">
+                  <icon name="font" class="text-dark" />
```

Comment:
Instead of using `font` which we already use for size options, I propose to use `sun` and `moon`.